deps/fast_float/fast_float_strtod.cpp

```diff
@@ -0,0 +1,38 @@
+#include "fast_float/fast_float.h"
```

Comment:
This file needs copyright information.

Reply:
Is this file from the `fast_float` project or is it our glue?

If it's our own glue code, I think we should put it under src/ rather than under deps/.

The deps files should be files from the dependency itself. I hope we don't have to modify them all to insert a copyright notice. If they don't have one, then I hope it's enough to include their LICENSE file.

Reply:
This one looks not from the original repo.

Reply:
Added license to glue to code.